index.html

```diff
@@ -0,0 +1,123 @@
+  integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+  crossorigin="anonymous"
+/>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    
+  <div id="pagina">
+    <header id="header">
+      <div id="header-img">
+        <img src="https://picsum.photos/150/100" alt="Logo">
+      </div>
+      <nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#home">Home</a></li>
+          <li ><a class="nav-link" href="#about">About</a></li>
+          <li><a class="nav-link" href="#pricing">Pricing</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="container">
+
+    <section id="hero">
+      <h2>Make your Bussines Grow!</h2>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input
+        name="email"
+        id="email"
+        type="email"
+        placeholder="Enter your email address"
+        required
+      />
+      <input id="submit" type="submit" value="Get Started" class="btn" />
+      </form>
+    </section>
+    
+    
+      <section id="home">
+        <div class="row">
+          <div class="icon"><i class="fas fa-chart-line"></i></div>
+          <div class="body">
+            <h2>Improve your Company!</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. At asperiores quae qui enim eos dolor eaque repudiandae, facere odio, similique, excepturi libero distinctio architecto sapiente harum expedita nulla. Nemo, unde.</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="icon"><i class="fas fa-globe-americas"></i></div>
+          <div class="body">
+            <h2>Reach all the world with one clic!</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. At asperiores quae qui enim eos dolor eaque repudiandae, facere odio, similique, excepturi libero distinctio architecto sapiente harum expedita nulla. Nemo, unde.</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="icon"><i class="fas fa-address-card"></i></div>
+          <div class="body">
+            <h2>Share your Resume!</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. At asperiores quae qui enim eos dolor eaque repudiandae, facere odio, similique, excepturi libero distinctio architecto sapiente harum expedita nulla. Nemo, unde.</p>
+          </div>
+        </div>
+      </section>
+    
+
+    <section id="about">
+      <iframe id="video" width="560" height="315" src="https://www.youtube.com/embed/Fn2OfRGbxwc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </section>
+
+    <section id="pricing">
+      <div class="product">
```

Comment:
You should change every div **class="product"** for an article tag. It's going to make your code semantic appropriately, clean and much easy to read.
Thanks 😃